vue-app/src/views/TransactionSuccess.vue

```diff
@@ -112,6 +112,7 @@ export default class TransactionSuccess extends Vue {
   width: 100%;
   display: flex;
   justify-content: center;
+  opacity:0.8;
```

Comment:
Thanks for the contribution @Patil2099!. I think this opacity would be better suited for the .money class. I think the money and confetti which use that class are what is causing this readability issue. If we apply the opacity to them, we can maintain the background this way. I think this also only needs to be done in the desktop size as well.